basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java

```diff
@@ -94,4 +103,69 @@ public void verifyIsDeploymentEnabledInServerGroup(String serverGroup, String de
     public void verifyIsDeploymentDisabledInServerGroup(String serverGroup, String deploymentName) throws Exception {
         verifyDeployementEnabledInServerGroup(serverGroup, deploymentName, false);
     }
+
+    public void verifyDeploymentContentDefault(List<String> itemsInDeploment) {
+        assertTrue(itemInListExistsLowerCase(itemsInDeploment, "meta-inf/manifest.mf | 127 b"));
+        assertTrue(itemInListExistsLowerCase(itemsInDeploment, "index.jsp | 361 b"));
+        assertTrue(itemInListExistsLowerCase(itemsInDeploment, "page.html | 68 b"));
+    }
+
+    public void enabledDisabledDeployment(WebDriver browser, FinderNavigation navigation, String deploymentName) {
+        Row row = navigation.step(FinderNames.DEPLOYMENT, deploymentName).selectRow();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        row.invoke(FinderNames.ENABLE);
+        Console.withBrowser(browser).openedWindow(ConfirmationWindow.class).confirm();
+    }
+
+    public void disableEnabledDeployment(WebDriver browser, FinderNavigation navigation, String deploymentName) {
+        Row row = navigation.step(FinderNames.DEPLOYMENT, deploymentName).selectRow();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        row.invoke(FinderNames.DISABLE);
+        Console.withBrowser(browser).openedWindow(ConfirmationWindow.class).confirm();
+    }
+
+    public void viewDeployment(WebDriver browser, FinderNavigation navigation, String deploymentName) {
+        Row row = navigation.step(FinderNames.DEPLOYMENT, deploymentName).selectRow();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        row.invoke(FinderNames.VIEW);
+    }
+
+    protected boolean deploymentIsEnabled(String introText) {
+        return introText.toLowerCase().contains("deployment is enabled");
+    }
+
+    protected boolean deploymentIsManaged(List<String> previewLines) {
+        return itemInListExistsLowerCase(previewLines, "managed: yes");
+    }
+
+    protected boolean deploymentIsArchive(List<String> previewLines) {
+        return itemInListExistsLowerCase(previewLines, "archive: yes");
+    }
+
+    protected boolean deploymentNeverEnabled(List<String> previewListItems) {
+        return itemInListExistsLowerCase(previewListItems, "the deployment was never enabled");
+    }
+
+    protected boolean deploymentNeverDisabled(List<String> previewListItems) {
+        return itemInListExistsLowerCase(previewListItems, "the deployment was never disabled");
+    }
+
+    public boolean deplomentEnabledTimestampExists(List<String> previewListItems) {
+        return itemInListExistsLowerCase(previewListItems, "last enabled at");
+    }
+
+    public boolean deplomentDisabledTimestampExists(List<String> previewListItems) {
+        return itemInListExistsLowerCase(previewListItems, "last disabled at");
+    }
+
+    protected boolean itemInListExistsLowerCase(List<String> lines, String item) {
```

Comment:
Hm, I am not sure why I let it as protected. I should make it public to ne consistet with other methods in class.